cert/lib/cert/commands_generator.rb

```diff
@@ -17,7 +17,7 @@ def run
       program :description, 'CLI for \'cert\' - Create new iOS code signing certificates'
       program :help, 'Author', 'Felix Krause <cert@krausefx.com>'
       program :help, 'Website', 'https://fastlane.tools'
-      program :help, 'GitHub', 'https://github.com/fastlane/fastlane/tree/master/cert#readme'
+      program :help, 'GitHub', 'https://docs.fastlane.tools/actions/cert/'
```

Comment:
`Documentation` instead of `GitHub` here, and in the other similar places.

Reply:
Done.